index.html

```diff
@@ -0,0 +1,169 @@
+                            <p class="card-text">With individual temperature settings that automatically adjust, drivers and front-seat passengers can customize their environment.</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="bg-light" id="members">
+                <div class="container">
+                    <div class="row">
+                        <div class="col-lg-12 text-center">
+                            <h2 class="section-heading text-dark">Featured Members</h2>
+                            <hr class="my-4">
+                        </div>
+                    </div>
+                    <div class="card-group" data-card-section></div>
+                </div>
+        </section>
+        <section id="contact" class="bg-secondary">
+            <div class="container">
+                <div class="row">
+                    <div class="col-lg-8 text-center mx-auto">
+                        <h2 class="section-heading text-white">Partners</h2>
+                        <hr class="my-4">
+                        <ul class="list partner-list">
+                            <li class="partner-item">
+                                <img src="https://pluspng.com/img-png/airbnb-logo-png-open-2000.png" alt="Airbnb Logo PNG"/>
+                            </li>
+                            <li class="partner-item">
+                                <img src="https://pluspng.com/img-png/airbnb-logo-png-open-2000.png" alt="Airbnb Logo PNG"/>
+                            </li>
+                            <li class="partner-item">
+                                <img src="https://pluspng.com/img-png/airbnb-logo-png-open-2000.png" alt="Airbnb Logo PNG"/>
+                            </li>
+                            <li class="partner-item">
+                                <img src="https://pluspng.com/img-png/airbnb-logo-png-open-2000.png" alt="Airbnb Logo PNG"/>
+                            </li>
+                            <li class="partner-item">
+                                <img src="https://pluspng.com/img-png/airbnb-logo-png-open-2000.png" alt="Airbnb Logo PNG"/>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="row"></div>
+            </div>
+        </section>
+    </main>
+    <footer class="bg-light">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-4" id="cc-commons">
+                    <img src="assets/img/logo/cc.logo.black.svg" alt="Creative commons logo">
+                </div>
+                <div class="col-lg-4 text-align-left" id="footer-text">
+                    <p>2022 Creative Commons Nairobi, Kenya.</p>
+                    <p>Some rights reverserd.</p>
+                </div>
+            </div>
+        </div>
+    </footer>
```

Comment:
- [ ] Kindly as requested by the previous reviewer, make you footer match the original design by make the two parts next to each other not above each other. On Home and About pages.
 
Your design
![image](https://user-images.githubusercontent.com/35535130/164465295-3ff5b5ad-65f7-468c-9781-65da854e9c0c.png)

Original design
![image](https://user-images.githubusercontent.com/35535130/164465241-d0999a25-ea0a-42a6-a57c-47ee104cb860.png)

Reply:
- Fixed section to design spec.
<img width="671" alt="Screenshot 2022-04-21 at 17 34 38" src="https://user-images.githubusercontent.com/40231363/164481872-0fe8ee0a-f9b5-444f-8a44-90e0bc81985d.png">